cishouseholds/weights/pre_calibration.py

```diff
@@ -22,12 +22,21 @@ def pre_calibration_high_level(df_survey: DataFrame, df_dweights: DataFrame, df_
         on="ons_household_id",
         how="left",
     )
-    # TEMPORARY
-    for column_name in ["ever_never_swab", "ever_never_antibodies", "longcovid", "14_days", "28_days", "42_days"]:
-        df = df.withColumn(column_name, F.lit(1))
-    df = df.withColumn("index_multiple_deprivation", F.col("index_multiple_deprivation").cast("float"))
-    # TODO - define logic for ever_never_swab, longcovid, 14_days, etc.
-    # TODO - extract index_multiple_deprivation, region_code columns
+
+    # TODO: find the right column name for each of the inputs
+    df = dataset_generation(
+        df=df,
+        # these cutoff dates need to be provided
+        cutoff_date_swab="2022-02-09",
+        cutoff_date_antibodies="2022-03-01",
+        cutoff_date_longcovid="2022-02-20",
+        column_test_result_swab="swab_result",
+        column_test_result_antibodies="antibodies_result",
+        column_test_result_longcovid="long_covid_have_symptoms",
+        patient_id_column="participant_id",
+        visit_date_column="visit_date",
```

Comment:
This should be either `visit_datetime` or `visit_date_string`, depending on whether time is needed